fabric-rendering-v1/src/client/java/net/fabricmc/fabric/mixin/client/rendering/BipedEntityRendererMixin.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.mixin.client.rendering;
+
+import com.llamalad7.mixinextras.injector.ModifyExpressionValue;
+import com.llamalad7.mixinextras.sugar.Local;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+
+import net.minecraft.client.render.entity.BipedEntityRenderer;
+import net.minecraft.item.ItemStack;
+
+import net.fabricmc.fabric.impl.client.rendering.ArmorRendererRegistryImpl;
+
+@Mixin(BipedEntityRenderer.class)
+abstract class BipedEntityRendererMixin {
+	@ModifyExpressionValue(method = "getEquippedStack", at = @At(value = "INVOKE", target = "Lnet/minecraft/client/render/entity/feature/ArmorFeatureRenderer;hasModel(Lnet/minecraft/item/ItemStack;Lnet/minecraft/entity/EquipmentSlot;)Z"))
+	private static boolean permitArmorWithCustomRenderers(boolean hasModel, @Local ItemStack stack) {
+		return hasModel || ArmorRendererRegistryImpl.get(stack.getItem()) != null;
```

Comment:
Question: Is there a change in behaviour if an item previously had an armor renderer for a given item?